src/guides/v2.3/config-guide/prod/config-reference-envphp.md

```diff
@@ -61,6 +62,18 @@ All the Magento cache types configuration are available from this node.
 
 Learn more about different [Cache Types][cache-types].
 
+## cron
+
+This node is used to enable/disable the cron jobs. `0` means disabled and `1` means enabled. If disabled, jobs will not run.
+
+```conf
+'cron' => [
+  'enabled' => 0
+]
+```
+
+Learn more about [Crons][crons].
+
```

Comment:
Added the reference to the warning note in previous suggestion, so you can remove it here.

```suggestion

```